lib/index.js

```diff
@@ -0,0 +1,27 @@
+var _ = require('lodash');
+
+var permissions = [];
+
+function addPermissions(permission) {
+    if (permissions.indexOf(permission) === -1) {
+        permissions.push(permission);
+    }
+    return this;
+}
+module.exports = {
+    addPermissions: function () {
+        var params = arguments;
+        if (arguments.length === 1 && arguments[0] instanceof Array) {
+            params = arguments[0];
+        }
+        _.forEach(params, addPermissions);
```

Comment:
I think we don't need to throw errors on duplicates, just ignore the dups.
btw. `_.prototype.concat` allows duplicates, that's why i do `indexOf`.